### DISCUSSION TITLE
CSS not loading

### DISCUSSION THREAD

**Discussion by sameerkumarg508** [community/community#76871] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

the css code is not loading on 'https://sameerkumarg508.github.io/' , though it is loading on my local machine.Following is my repository, https://github.com/sameerkumarg508/sameerkumarg508.github.io/, can someone check this issue please

**Comment by guspan-tanadi** [comment]
Hi, looks like styles.css is loaded here using [Chromium](https://github.com/chromium/chromium) browser.

Is it the layout much like screenshot below?

![Screenshot from 2023-11-25 12-17-47](https://github.com/community/community/assets/36249910/0fb88724-9b0e-4bc9-93ec-172a2423741d)

**Reply by sameerkumarg508** [reply]
Hi Guspan, my issue is already fixed as i have worked through it yesterday. Anyways thanks for trying to help me on my issue

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sameerkumarg508 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.
